.travis.yml

```diff
@@ -0,0 +1,32 @@
+sudo: false
```

Comment:
`sudo:` no longer has any effect: https://blog.travis-ci.com/2018-11-19-required-linux-infrastructure-migration

```suggestion
```

Reply:
Thanks for all these comments Hugo! Will definitely these 👍